src/core/remote.js

```diff
@@ -1822,6 +1835,10 @@ Remote.prototype.findAccount = function(accountID) {
  */
 Remote.prototype.createPathFind = function(options, callback) {
   if (this._cur_path_find !== null) {
+    if (callback === undefined) {
```

Comment:
You mean is to create new `PathFind`, set it to `_cur_path_find` and return it?
The only problem with that, is what in `this._queued_path_finds` can be left hanging some requests.